BraveRewardsUI/Settings/Tips/Details/TipsDetailViewController.swift

```diff
@@ -36,6 +38,41 @@ class TipsDetailViewController: UIViewController {
     tipsView.tableView.dataSource = self
 
     title = Strings.Tips
+    loadData()
+  }
+  
+  private var nextContributionDateView: LabelAccessoryView {
+    let view = LabelAccessoryView()
+    let dateFormatter = DateFormatter().then {
+      $0.dateStyle = .short
+      $0.timeStyle = .none
+    }
+    let reconcileDate = Date(timeIntervalSince1970: TimeInterval(state.ledger.autoContributeProps.reconcileStamp))
+    view.label.text = dateFormatter.string(from: reconcileDate)
```

Comment:
This is still not using the `Date.stringFrom` API?

Reply:
Ah sorry, copied it from you old code to computed property and forgot to change it. Updating